apps/predbat/config/apps.yaml

```diff
@@ -226,6 +226,10 @@ pred_bat:
   # Standing charge in pounds, can be set to a sensor or manually entered (e.g. 0.50 is 50p)
   # The default below will pick up the standing charge from the Octopus Plugin
   metric_standing_charge: 're:(sensor.(octopus_energy_|)electricity_[0-9a-z]+_[0-9a-z]+_current_standing_charge)'
+  #
+  # If you don't want to include the standing charge in batpred's cost prediction, set metric_standing_charge to zero.
```

Comment:
Or just delete the line

Reply:
True, either way, I thought it worthwhile adding an explanation of this as I personally prefer to just focus on the consumption. I can't do anything about the standing charge and it was confusing me to include it

Reply:
I've added some text about this on main now